src/components/application_manager/include/application_manager/policies/policy_handler.h

```diff
@@ -685,6 +685,8 @@ class PolicyHandler : public PolicyHandlerInterface,
 
   virtual void OnPTInited() OVERRIDE;
 
+  void ForceRetrySequenceStop() OVERRIDE;
```

Comment:
I would name it `ForceStopRetrySequence`

Reply:
@IGapchuk I would name it more simply : `StopRetrySequence`

Reply:
@mked-luxoft @AByzhynar Has been renamed to `StopRetrySequence`. Please, take a look at 847ef5be3d621cabf18d1f8c46810c8443a428cf